Program.cs

```diff
@@ -1,10 +1,14 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿//Write a program that reverses a given number and check if it is a palindrome. 
+//Note: The input should first be converted to an integer before the check.
+//Do not do any string manipulations like reversing a string and converting it to an integer.
+
+Console.Write ("Enter a number: ");

```

Comment:
Fix this error.
![image](https://github.com/RaaneshKumar/Training/assets/94427955/896687c8-8031-4331-95d2-e8bd2b19aab2)